src/test/java/seedu/address/model/person/HourlySalaryTest.java

```diff
@@ -31,20 +31,31 @@ public void constructor_alphanumericSalary_throwsIllegalArgumentException() {
     }
 
     @Test
-    public void isValidSalary() {
+    public void isValidHourlySalary() {
         // null hourlySalary
-        assertThrows(NullPointerException.class, () -> HourlySalary.isValidSalary(null));
+        assertThrows(NullPointerException.class, () -> HourlySalary.isValidHourlySalary(null));
 
-        // invalid hourlySalary
-        assertFalse(HourlySalary.isValidSalary("")); // empty string
-        assertFalse(HourlySalary.isValidSalary(" ")); // spaces only
-        assertFalse(HourlySalary.isValidSalary("^")); // only non-alphanumeric characters
-        assertFalse(HourlySalary.isValidSalary("power-ranger")); // contains non-numeric characters only
-        assertFalse(HourlySalary.isValidSalary("1.123m")); // contains alphanumeric characters
-        assertFalse(HourlySalary.isValidSalary("-1.10")); // contains negative values
+        // invalid hourlySalary from formatting
+        assertFalse(HourlySalary.isValidHourlySalary("")); // empty string
+        assertFalse(HourlySalary.isValidHourlySalary(" ")); // spaces only
+        assertFalse(HourlySalary.isValidHourlySalary("^")); // only non-alphanumeric characters
+        assertFalse(HourlySalary.isValidHourlySalary("power-ranger")); // contains non-numeric characters only
+        assertFalse(HourlySalary.isValidHourlySalary("1.123m")); // contains alphanumeric characters
+        assertFalse(HourlySalary.isValidHourlySalary("-0.00")); // contains negative values
+        assertFalse(HourlySalary.isValidHourlySalary("0.001")); // contains more than 2 decimal places
 
         // valid hourlySalary
```

Comment:
Very minor nitpick, but maybe a more descriptive comment here would be "valid values in equivalence partition [0.00, 1000.00]" to fit with the format of the other comments?

Reply:
I think that does sound better! I have updated both HourlySalaryTest and OvertimePayRateTest regarding this, thanks!